### DISCUSSION TITLE
Vase Mode + Height Range Modifier

### DISCUSSION THREAD

**Discussion by JonBerlin** [OrcaSlicer/OrcaSlicer#3621] [Ideas]
First and foremost, a big thanks to Andrew @andrewboktor for fixing the vase mode in v1.9. It was a long time coming (10 years!).

Another feature request related to spiral vase mode, which has not be implemented in any Slic3r fork for years, is the option to use **spiral vase mode in combination with the height range modifier**.

I frequently use self-made G-Code files that involve a combination of regular layer mode and a vase mode section in between. However, creating these hybrid G-Code files require a considerable effort.

This should be a quick fix because, in spiral vase mode, the first bottom shell layers are printed using the normal layer mode, and the layer structure is also maintained in spiral vase mode.

**Comment by guategeek** [comment]
I also would like this feature.

**Comment by CraigHutchinson** [comment]
This would be a superb feature and something I was hoping was already possible.

**Comment by Aurasphere-au** [comment]
prob needs to  be merged with 
https://github.com/SoftFever/OrcaSlicer/discussions/7027

**Comment by petertrotman** [comment]
Yes please to this, dealing with this issue right now.

**Comment by Adqam64** [comment]
Echoing this - it would be useful for various tests if this worked.
